src/main/riddl/Organization/organization.riddl

```diff
@@ -30,66 +49,326 @@ context OrganizationContext is {
                 }
                 on command EditOrganizationInfo {
                     then set Draft.info to @EstablishOrganization.info
-                    // and yield event OrganizationInfoUpdated(orgId=@Draft.orgId, orgInfo=@Draft.orgInfo, orgMetaInfo=@Draft.orgMetaInfo)
-                    //TODO: turn this into a function and call function here
+                    and yield event OrganizationInfoUpdated(orgId=@Draft.orgId, info=@Draft.orgInfo, meta=@Draft.orgMetaInfo)
+                }
+                on command UpdateParent {
+                    example Success is {
+                        then set Draft.parent to @EstablishOrganization.parent
```

Comment:
Here as well, shouldn't it read `@UpdateParent.parent`? You are doing the same thing in the other State handlers below too

Reply:
You are correct. I am replacing the above with a more descriptive form of command handling. i.e.

on command UpdateParent {
  then "set draft parent field to reflect new parent provided in UpdateParent command"
  and "yield event ParentUpdated"
}